Accepted-RFCs/RFC-011-news-announcements.md

```diff
@@ -0,0 +1,165 @@
+
+Heartbeats are periodic posts where we talk about the states of different working groups, as relevant to our wider audience of users or prospective contributors. These would be ideally no more than every two or three months, and drafted from a perspective of PR to keep showing in between release cycles that we're passionate about our work.
+
+Heartbeats could also possibly showcase an individual contributor, like a "meet the team" sort of vibe. This has been declined as a blog post topic, as that doesn't fit the purpose of the blog, but it could fit here at the tail end of heartbeat posts.
+
+(That also gives those individual contributors featured a different sort of call to action on publicizing their related post, as friends/colleagues of that contributor may be interest in the post for relationship reasons, then find the other content or project in general interesting enough to follow on its own.)
+
+#### What this isn't
+
+This wouldn't be a place for internal communications, meeting notes, etc. 
+
+This also wouldn't be a "second blog" or similar.
+
+### Why not use the blog?
+
+The blog is about "docs advocacy & education," so if peopple subscribe to the blog/look out for new posts, they should expect material that's education, not that's purely about our group marketing.
+
+The other elements of the blog, like its particular formatting/visual branding, focus on author, everything about CC licensing, etc. do not and should not apply to this content bucket.
+
+### Specifics
+
+The main page should show the most recent post, so when one clicks on "News" in the header, they don't have to click on a second link to see what's the most recent.
+
+The sidebar would have the most recent 5 or so posts, with a "see all" that would just show a list with short versions, akin to what is seen in the [Write the Docs newsletter archive page](https://www.writethedocs.org/blog/archive/tag/newsletter/).
+
+The posts should be short: 500 words or so. Something that would take on average less than 5 minutes to read. We may deviate from this as needed, but the core concept is that our newsletters shouldn't take significant resources to craft and review, and we shouldn't demand significant time expenses from our audience in this manner.
+
+Unlike how we handle the blog, authors wouldn't link to "about the author" pages as we do in the blog—and in fact we may decide there are no listed individual authors, to keep the news to appear to always be from the community.
+
+Unlike how we handle the blog, there wouldn't be any commenting feature. 
+
+Unlike how we handle the blog, there wouldn't be any banner or hero images.
+
+Because news is timely, the individual post URLs would use the date rather than a title.
```

Comment:
This would not allow more than 1 post a day, unless we also add the time to the url.  We could just generate an ISO datetime string for the slug.  ISO datetime would probably be easiest and would provide some information in the string that would mean something about when the post was created.

Reply:
That's a good point!
Ideally we don't want more than one news post a day, but to my earlier point about the potential need for a public apology (or something like an emergency rollback or whatever, should we for some reason need that—like if we get into doc tooling), we should make sure we can support multiple posts per day.

Better to support and never need than kick ourselves for not having it, right? As long as it doesn't turn into overengineering. (I'm okay to engineer for contingencies we need to support as the org grows, as long as they're reasonable to consider, and it's less costly to implement them now than later.)

Reply:
We have to name the files something, might as well be a date/time code if we aren't using text slugs.

Reply:
There's probably other valid reasons for multiple posts per day, e.g. what we're doing/have done during WTD conferences. I'd argue that just appending a "-2" or similar would be sufficient, but using a timestamp works too.